Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
```

Comment:
This line would suggest that operations are directed exclusively by the TSC, however, since the CommComm is on equal footing under the Board and also uses the GitHub repo, I feel like it should be listed here. I also wonder if that relationship is part of what this document is sussing out, and something we should further discuss!